src/ekore/operator_matrix_elements/polarized/space_like/as2.py

```diff
@@ -1,36 +1,33 @@
 r"""
-This module contains the |NNLO| |OME| in the polarized case for the matching conditions in the |VFNS|.
-The equations given in :cite:`Bierenbaum_2023`. In the paper the fraction :math:`\mu_F^2 / m_H^2` inside the log is inverted, yielding an additional factor of -1 wherever L has an odd power. The anomalous dimensions are expressed with an additional factor -1 as well relative to the paper. The anomalous dimensions with the addition 'hat', have the form :math:`\gamma_hat = gamma(nf+1) - gamma(nf)`.
+This module contains the |NNLO| OME's in the polarized case for the matching conditions in the |VFNS|.
+The equations are given in :cite:`Bierenbaum_2023`. As in the |NLO| OME's, in the paper, an additional factor 2 can be found in front of the anomalous dimensions and factor (-1) for odd powers of L. The anomalous dimensions and beta function with the addition 'hat' are defined as in the |NLO| case.
+
+
 """
```

Comment:
```suggestion
This module contains the |NNLO| |OME| in the polarized case for the matching conditions in the |VFNS|.

The equations are given in :cite:`Bierenbaum_2023`. As in the |NLO| |OME|, in the paper, an additional factor 2 can be found in front of the anomalous dimensions and factor (-1) for odd powers of L. The anomalous dimensions and beta function with the addition 'hat' are defined as in the |NLO| case.
"""
```
I'm almost sure, pre-commit will complain about these style hickups